src/components/Router/Router.tsx

```diff
@@ -3,7 +3,7 @@ import { BrowserRouter, HashRouter } from 'react-router-dom';
 import type { BrowserRouterProps, HashRouterProps } from 'react-router-dom';
 
 export default function Router(props: BrowserRouterProps | HashRouterProps) {
-  if (import.meta.env.SNOWPACK_PUBLIC_GITHUB_PAGES) {
+  if (import.meta.env.APP_PUBLIC_GITHUB_PAGES) {
```

Comment:
It seems, that if we are not in a build mode, this value equals to `undefined`. Maybe we should better define it in `vite-config` in `define` section?

Reply:
Does it cause a problem if it's undefined?  Both with the dev server and serving the static build files, it just evaluates to falsey for me without any errors or warnings.